src/app/body/filter-task/filter-task.component.ts

```diff
@@ -40,12 +47,63 @@ export class FilterTaskComponent implements OnInit {
   sprint: number
   showLoader: boolean = false;
 
-  constructor(public appSettings: ApplicationSettingsService, public backendService: BackendService) { }
+  constructor(public appSettings: ApplicationSettingsService, public backendService: BackendService, public startService: StartServiceService, public authService: AuthService) { }
 
   ngOnInit(): void {
-    this.project = this.defaultProject
-    this.onProjectChange();
+    if(this.startService.showTeamsData) {
+      this.project = this.authService.getTeamId();
+      this.teamIds = this.backendService.getOrganizationTeamIds();
+      this.readTeamData(this.project);
+      this.project = this.defaultProject
+      this.onProjectChange();
```

Comment:
What is the Use of Calling this Fuction again when we are calling readTeanData()